cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectAssemblyOperatorTest.java

```diff
@@ -91,6 +89,8 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
+// KafkaConnectS2I is deprecated
+@SuppressWarnings({"unchecked", "deprecation",  "checkstyle:ClassFanOutComplexity"})
```

Comment:
KafkaConnectS2I is gone now. So the deprecation should not be needed.

Reply:
Done :)